### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by lannisterrr**
Initial boilerplate added.

### CODE REVIEW

File: css/homepage.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
+.main-head.ecom-navbar {
+  background: var(--primary-color);
+  position: sticky;
+  top: 0px;
+  z-index: 3;
```
Comment (Hamdrive) [inline css/homepage.css]: Lateron there might be stacking context conflicts, best to keep navbar z-index at highest possible value like 9999

File: css/homepage.css [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+.category-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+}
+
+.category-wrapper {
+  cursor: pointer;
+  /* border: 2px solid var(--primary-color); */
```
Comment (Hamdrive) [inline css/homepage.css]: Clean up comments

**Review by Hamdrive — COMMENTED** [review]
Looks good, changes recommended

File: css/homepage.css [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+/* hero image */
+.hero {
+  padding: 2rem;
+  padding-top: 5rem;
+}
+
+.hero .hero-img {
+  max-width: 80%;
+  width: 550px;
```
Comment (aryan-13) [inline css/homepage.css]: Relative units are better for responsiveness

File: css/homepage.css [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+}
+
+.category-wrapper {
+  cursor: pointer;
+  /* border: 2px solid var(--primary-color); */
+  width: 217px;
+  height: 217px;
```
Comment (aryan-13) [inline css/homepage.css]: Here too

**Review by aryan-13 — COMMENTED** [review]
The code looks good to merge, after making the suggested changes!

**Pull request merged**